frontend/app/mocks/client-application-data/client-application.json

```diff
@@ -0,0 +1,267 @@
+                "TelephoneNumberFullID": "555-555-5555"
+              },
+              "TelephoneNumberCategoryCode": {
+                "ReferenceDataName": "Primary"
+              }
+            },
+            {
+              "FullTelephoneNumber": {
+                "TelephoneNumberFullID": "555-555-5555"
+              },
+              "TelephoneNumberCategoryCode": {
+                "ReferenceDataName": "Alternate"
+              }
+            }
+          ]
+        }
+      ],
+      "PersonLanguage": [
+        {
+          "CommunicationCategoryCode": {
+            "ReferenceDataID": ""
+          },
+          "PreferredIndicator": ""
+        }
+      ],
+      "PersonMaritalStatus": {
+        "StatusCode": {
+          "ReferenceDataID": "775170001"
+        }
+      },
+      "PersonName": [
+        {
+          "PersonGivenName": ["MyFirstName50"],
+          "PersonSurName": "MyLastName50"
+        }
+      ],
+      "PersonSINIdentification": {
+        "IdentificationID": "999999999",
+        "IdentificationCategoryText": "some text"
+      },
+      "MailingSameAsHomeIndicator": true,
+      "PreferredMethodCommunicationCode": {
+        "ReferenceDataID": "775170002"
+      },
+      "ApplicantDetail": {
+        "AttestParentOrGuardianIndicator": true,
+        "ConsentToSharePersonalInformationIndicator": true,
+        "DisabilityTaxCreditIndicator": true,
+        "FederalDentalCoverageIndicator": true,
+        "InsurancePlan": [
+          {
+            "InsurancePlanIdentification": [
+              {
+                "IdentificationID": "",
+                "IdentificationCategoryText": ""
+              },
+              {
+                "IdentificationID": "",
+                "IdentificationCategoryText": ""
+              }
+            ]
+          },
+          {
+            "InsurancePlanIdentification": [
+              {
+                "IdentificationID": "",
+                "IdentificationCategoryText": ""
+              },
+              {
+                "IdentificationID": "",
+                "IdentificationCategoryText": ""
+              }
+            ]
+          }
+        ],
+        "LivingIndependentlyIndicator": true,
+        "PrivateDentalInsuranceIndicator": true,
+        "ProvincialDentalCoverageIndicator": true
+      },
+      "ApplicantCategoryCode": {
+        "ReferenceDataID": "775170000",
+        "ReferenceDataName": ""
+      },
+      "ClientIdentification": [
+        {
+          "IdentificationID": "4f35f70b-2f83-ee11-8179-000d3a09d000",
+          "IdentificationCategoryText": "Applicant ID"
+        },
+        {
+          "IdentificationID": "1e97fe42-0263-ee11-8df0-000d3a09df08",
+          "IdentificationCategoryText": "Client ID"
+        }
+      ],
+      "Flags": [
+        {
+          "Flag": false,
+          "FlagCategoryText": "isCraAssessed"
+        },
+        {
+          "Flag": true,
+          "FlagCategoryText": ""
+        }
```

Comment:
It might be worth adding the flag that indicates if the client applied before April 30, 2024. This flag will be used to distinguish whether the client enters the ITA flow or the "Adult or family" flow for renewal.